docs/content/ruby/guides/data_filtering_next/index.md

```diff
@@ -1,104 +1,68 @@
 ---
-title: "Filter Data - Adapter API (Preview)"
+title: "Filter Data (Preview)"
 weight: 1
 showContentForAnyLanguage: true
-no_nav: true
```

Comment:
I added this intentionally so that the page is only accessible from Ruby data filtering documentation, rather than adding a second "Filter Data" entry at the top level.